### ISSUE TITLE
Info node unavailable from the top info node when installing from melpa

### ISSUE THREAD

**Issue by aagon** [joostkremers/ebib#189]
Expected behaviour : Open top info node (```M-x info```), find the ebib node inside it.

Actual behaviour : No ebib node.

Steps to reproduce :
- Install ebib from melpa (or melpa-stable) : ```(use-package ebib :ensure t)```
- No ebib node

This probably comes from there being no dir file inside the relevant directory (```~/.emacs.d/elpa/ebib-20201223.2217/```) to reference the node. Adding such a file solves the problem.

After verification, melpa packages that distribute an info node all provide the dir file that references said info node. I don't know whether the problem comes from your end or from the melpa recipe ; I didn't provide a PR for this reason.

Best,

Aymeric Agon-Rambosson

**Comment by joostkremers** [comment]
I removed the `dir` file from the git repo a couple of years ago because it's supposed to be created automatically (see #117). I've looked at three Melpa packages that appear in my `info` listing (burly, dash, org-roam), and they don't have a `dir` file in their repos that I can see.

Still, installing `ebib` doesn't seem to create a `dir` file. I don't know what's going on there, I'll have to figure that out.
